### ISSUE TITLE
5.3 Update

### ISSUE THREAD

**Issue by Ben-Garcia-DB-Solutions** [W2Wizard/SimpleColorWheel#4]
Any chance of an update to Unreal 5.3? Lush plugin.

**Comment by W2Wizard** [comment]
Here we go again ...
I'll see what needs to be done.

**Comment by Ben-Garcia-DB-Solutions** [comment]
Appreciate it.

I honestly have very little idea around the C++ side of Unreal but I downloaded the plugin & just changed the version to 5.3 & at first glance it seemed to work fine.

Please visit www.dbsolutions.com
d&b solutions UK Ltd. Registered in England No. 01031687
Registered Office: 20 Merton Industrial Park, Jubilee Way, Wimbledon, SW19 3WL. VAT No: GB 234 0411 15

**Comment by W2Wizard** [comment]
Yeah thats my problem with this, I shouldn't have to manually send a file to epic every time a minor version comes out. Rendering the plugin useless but still technically totally fine.

But alas I have no choice, personally I always use plugins from source, less of a hassle to manage.

**Comment by Ben-Garcia-DB-Solutions** [comment]
Yeah, plugin users should absolutely be able to just say "I don't care this is 1/2/23 minor versions out of date, I'll try it!".

To be honest, I don't know how to build plugins or the engine from source. 1 person operation & I have absolutely no C++ skill so I don't want to open that can of worms!

Please visit www.dbsolutions.com
d&b solutions UK Ltd. Registered in England No. 01031687
Registered Office: 20 Merton Industrial Park, Jubilee Way, Wimbledon, SW19 3WL. VAT No: GB 234 0411 15


Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ackys** [comment]
I just compiled this for UE 5.3 without problems. Thank you.

**Comment by JoshQuake** [comment]
Hmm I changed the engine version to 5.3 in the uplugin but it still fails to build for me.
